.gitignore

```diff
@@ -0,0 +1,2 @@
+.vagrant/
+desktop.ini
```

Comment:
See the "no-newline at the end of file" warning above - please fix that - there should be an appropriate setting for your text editor.